package.json

```diff
@@ -15,7 +15,7 @@
     "test-watch": "jest --watch",
     "clean": "rimraf dist/* && mkdirp dist",
     "codegen": "run-s codegen:*",
-    "codegen:actorCids": "rimraf src/generated/actorCodes.ts && mkdir -p src/generated && touch src/generated/actorCodes.ts && node src/scripts/generate_actor_codes.mjs && eslint ./src/generated/actorCodes.ts --fix",
+    "codegen:actorCids": "node src/scripts/generate_actor_codes.mjs && eslint ./src/generated/actorCodes.ts --fix",
```

Comment:
Removes commands that only work on a Unix system. Rimraf works because it's installed as a node module, but there shouldn't be any need because the `actorCodes.ts` file is overwritten by the script